ZFSin/zfs/module/zfs/metaslab.c

```diff
@@ -1610,6 +1639,12 @@ metaslab_load(metaslab_t *msp)
 	if (msp->ms_loaded)
 		return (0);
 	VERIFY(!msp->ms_loading);
+	ksp = msp->ms_group->mg_kstat;
+	if (ksp != NULL) {
```

Comment:
Do you need to check for `if (msp->ms_group != NULL)` before this, similar to what you did for `metaslab_unload()`?

Reply:
added null check